src/legacy/core_plugins/kibana/public/dashboard/np_ready/legacy_app.js

```diff
@@ -207,7 +212,7 @@ export function initDashboardApp(app, deps) {
               .catch(error => {
                 // A corrupt dashboard was detected (e.g. with invalid JSON properties)
                 if (error instanceof InvalidJSONProperty) {
-                  deps.toastNotifications.addDanger(error.message);
+                  deps.core.notifications.toasts.addDanger(error.message);
```

Comment:
Fix: `deps.toastNotifications` does not exist

Reply:
Great catch!